annif/cli.py

```diff
@@ -83,6 +83,30 @@ def open_doc_path(path, subject_index):
     return docs
 
 
+def open_text_documents(paths, docs_limit):
```

Comment:
This adds yet another utility function to the top of cli.py. Nothing wrong with that, but cli.py has grown very long so I think we could refactor this in a follow-up PR, moving the utility functions to a separate module such as cli_util.py. Then cli.py would just contain the Click-decorated functions that implement the CLI commands themselves.